iot-e2e-tests/common/pom.xml

```diff
@@ -147,7 +147,7 @@
                     <forkCount>1</forkCount>
 
                     <!--Tests can be run in parallel with tests within their own test class as well as in parallel with tests from other test classes-->
-                    <parallel>both</parallel>
+                    <parallel>classes</parallel>
```

Comment:
"both" is deprecated now, it basically means both methods and class tests run in parallel

Reply:
But with this change I am deleting methods meaning tests within a class will run sequentially.